twilight-model/src/channel/message/component/mod.rs

```diff
@@ -29,28 +28,104 @@ use serde::{
 use serde_value::{DeserializerError, Value};
 use std::fmt::{Formatter, Result as FmtResult};
 
-/// Interactive element of a message that an application uses.
+/// Interactive message element.
+///
+/// Must be either a top level [`ActionRow`] or nested inside one.
+///
+/// # Examples
+///
+/// Button
+///
+/// ```
+/// use twilight_model::channel::message::component::{ActionRow, Button, ButtonStyle, Component};
+///
+/// Component::ActionRow(ActionRow {
+///     components: vec![Component::Button(Button {
```

Comment:
should use `Vec::new` here

Reply:
I personally prefer examples having idiomatic code style, overruling Twilight's preferences.